### PR TITLE
fix(DB/Creatures): Added waypoint script to Black Stalker.

### LINKED ISSUE #13794: [The Underbog] The Black Stalker missing scripted waypoints with emotes
### Current Behaviour

Static spawn

### Expected Blizzlike Behaviour

Moves around. After reaching certain points, emotes for ~12s and then resumes movement.

### Source

https://youtu.be/I16uIzEBQe8?t=1737

### Steps to reproduce the problem

1. `.go xyz 200.5 7 27.562534 546 3`
2. Observe the boss.

### Extra Notes
See my PR for a re-constructed Underbog: https://github.com/azerothcore/azerothcore-wotlk/pull/13769

Emote: 
```
ServerToClient: SMSG_EMOTE (0x27C9) Length: 27 ConnIdx: 1 Time: 10/31/2022 23:47:40.065 Number: 25676
GUID: Full: 0x204CB04440117680000146000060812E Creature/0 R4908/S326 Map: 546 Entry: 17882 Low: 6324526
Emote ID: 7 (OneShotEat)
SpellVisualKitCount: 0
SequenceVariation: 0
```

Waypoints:
```
-- The Black Stalker
SET @NPC := XXXXXX;
SET @PATH := @NPC * 10;
UPDATE `creature` SET `wander_distance`=0,`MovementType`=2 WHERE `guid`=@NPC;
DELETE FROM `creature_addon` WHERE `guid`=@NPC;
INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`visibilityDistanceType`,`auras`) VALUES (@NPC,@PATH,0,0,1,0,0, '');
DELETE FROM `waypoint_data` WHERE `id`=@PATH;
INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
(@PATH,1,143.12927,24.530138,26.791546,NULL,0,0,0,100,0),
(@PATH,2,147.65775,5.377345,26.791546,NULL,0,0,0,100,0),
(@PATH,3,137.54929,3.156878,26.791546,NULL,0,0,0,100,0),
(@PATH,4,147.90135,15.377421,26.796476,NULL,0,0,0,100,0),
(@PATH,5,151.38458,20.670511,26.857647,NULL,0,0,0,100,0),
(@PATH,6,143.12927,24.530138,26.791546,NULL,0,0,0,100,0),
(@PATH,7,141.65916,32.419727,26.831083,NULL,0,0,0,100,0),
(@PATH,8,163.50163,11.745237,26.72663,NULL,0,0,0,100,0),
(@PATH,9,168.98875,8.392987,26.64813,NULL,0,0,0,100,0),
(@PATH,10,147.65775,5.377345,26.791546,NULL,0,0,0,100,0),
(@PATH,11,137.54929,3.156878,26.791546,NULL,0,0,0,100,0),
(@PATH,12,147.90135,15.377421,26.796476,NULL,0,0,0,100,0),
(@PATH,13,151.38458,20.670511,26.857647,NULL,0,0,0,100,0);
```

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/fb08f8e905b92029040017c24198d9e6008a2c5a

### Operating system

Windows 10

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #13794

## Issues Addressed:

- Closes #13794

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

`.go xyz 200.5 7 27.562534 546 3`
Observe the boss.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Gultask** [comment]
Looks better with delay 2 and datalong2 set to 0, in my opinion. The emote doesn't repeat in the video.

**Comment by Gultask** [comment]
By the way, is the wiki outdated?

image

**Comment by Nyeriah** [comment]
> By the way, is the wiki outdated?
> 
> image

It's not. It's used by that godawful wp show command. It links the whisps to the waypoint.

File: data/sql/updates/pending_db_world/rev_1668944605510491900.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+(@CGUID*10,8,163.50163,11.745237,26.72663,0,0,0,0,100,0),
+(@CGUID*10,9,168.98875,8.392987,26.64813,NULL,15000,0,@CGUID*10,100,@CGUID), 
+(@CGUID*10,10,147.65775,5.377345,26.791546,0,0,0,0,100,0),
+(@CGUID*10,11,137.54929,3.156878,26.791546,NULL,15000,0,@CGUID*10,100,@CGUID),
+(@CGUID*10,12,147.90135,15.377421,26.796476,0,0,0,0,100,0),
+(@CGUID*10,13,151.38458,20.670511,26.857647,NULL,15000,0,@CGUID*10,100,@CGUID);
+
+DELETE FROM `waypoint_scripts` WHERE `id`=@CGUID*10;
+INSERT INTO `waypoint_scripts` VALUES
+(@CGUID*10,1,1,7,1,0,0,0,0,0,@CGUID);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1668944605510491900.sql:27]:
```suggestion
(@CGUID*10,2,1,7,0,0,0,0,0,0,@CGUID);
```
@UltraNix Are you going to change it?

Comment (UltraNix) [reply data/sql/updates/pending_db_world/rev_1668944605510491900.sql:27]: No? It's okay. 1 as it's emote and 1 as it's a emote state.

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1668944605510491900.sql:27]: I meant changing the delay to 2 and datalong to 0. The emote only plays once in the video.

**Pull request merged**